AdobeIms/Model/UserProfile.php

```diff
@@ -65,6 +65,22 @@ public function getEmail(): ?string
         return $this->getData(self::EMAIL);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function getAvatar(): ?string
```

Comment:
I think it's better to change the terminology from `Avatar` to `Image` everywhere in the code

Reply:
ok